dadi/lib/cache/index.js

```diff
@@ -189,7 +189,7 @@ Cache.prototype.init = function () {
 
     // we build the filename with a hashed hex string so we can be unique
     // and avoid using file system reserved characters in the name
-    const requestUrl = url.parse(req.url, true).path
+    const requestUrl = req.url
```

Comment:
My only comment would be that `requestUrl` now becomes a bit redundant (and actually longer in length than just `req.url`), but I'm not strongly against it.